arcgis-ios-sdk-samples/Route & Directions/Route around barriers/README.md

```diff
@@ -1,20 +1,73 @@
+In some situations, it is further beneficial to find the most efficient route that reaches all stops, reordering them to reduce travel time. For example, a delivery service may target a number of drop-off addresses, specifically looking to avoid congested areas or closed roads, arranging the stops in the most time-effective order.
+
+## How to use the sample
 
-![](image1.png)
-![](image2.png)
+Tap 'Add stop' to add stops to the route. Tap 'Add barrier' to add areas that can't be crossed by the route. Tap 'Route' to find the route and display it. Tap the settings button to toggle preferences like find the best sequence or preserve the first or last stop. Additionally, tap the directions button to view a list of the directions. 
 
 ## How it works
 
-The sample uses the `defaultRouteParameters(completion:)` method on `AGSRouteTask` to get the default parameters from the service. Barriers are created using the initializer `init(polygon:)` on `AGSPolygonBarrier`. Sets the stops and barriers in those parameters and the `returnDirections` to true. Then uses the `solveRoute(with:completion:)` method to solve for the route. Once the route is in, the sample displays the list of directions using the `directionManeuvers` property on the `AGSRoute` returned in the `AGSRouteResult`.
+1. Create the route task by calling `AGSRouteTask(url:)` with the URL to a Network Analysis route service.
+2. Get the default route parameters for the service by calling `AGSRouteTask.getDefaultParameters`.
+3. When the user adds a stop, add it to the route parameters.
+    i. Normalize the geometry; otherwise the route job would fail if the user included any stops over the 180th degree meridian.
+    ii. Get the name of the stop by counting the existing stops - `.stopGraphicsOverlay.graphics.index(of:) + 1`.
+    iii. Create a composite symbol for the stop. This sample uses a blue marker and a text symbol.
+    iv. Create the graphic from the geometry and the symbol.
+    v. Add the graphic to the stops graphics overlay.
+4. When the user adds a barrier, create a polygon barrier and add it to the route parameters.
+    i. Normalize the geometry (see **3i** above).
+    ii. Buffer the geometry to create a larger barrier from the tapped point by calling `AGSGeometryEngine.bufferGeometry(geometry:byDistance:)`.
```

Comment:
```suggestion
    ii. Buffer the geometry to create a larger barrier from the tapped point by calling `class AGSGeometryEngine.bufferGeometry(geometry:byDistance:)`.
```
Static methods should include the `class` part, and it will be copied if you use the copy qualified symbol name shortcut mentioned above.